src/runtime/components/forms/SelectMenu.vue

```diff
@@ -355,17 +359,34 @@ export default defineComponent({
       }
     })
 
+    const selected = computed(() => {
```

Comment:
Not sure to understand why you added this computed, could you provide a real-life example where you need this?

Reply:
Sure!
When you change the selected value using the `value-attribute` prop, there is currently no way (at least as far as I'm aware) to access the remaining attributes of the selected object.
Let's say we have an array of categories with translated names:
```ts
const categories = [
  {
    id: 1,
    name: {
      en: 'Category',
      de: 'Kategorie',
      // ...
    },
  },
  // ...
]
```

This is how I would use it:
```vue
<USelectMenu
    :options="categories"
    value-attribute="id"
>
    <template #label="{ selected }">
        {{ useLocalizedValue(selected.name) }}
    </template>
</USelectMenu>
```

This applies to the `#leading` and `#trailing` slots as well. Currently, you wouldn't be able to access any icon attributes of the object in those slots, if you changed the `value-attribute`, would you? So it might be worth adding this to those slots as well.
```vue
<!--  example from the docs  -->
<template>
  <USelectMenu v-model="selected" :options="people"> <!--  if we add `value-attribute="id"`, this won't work  --> 
    <template #leading>
      <UIcon v-if="selected.icon" :name="(selected.icon as string)" class="w-4 h-4 mx-0.5" />
      <UAvatar v-else-if="selected.avatar" v-bind="(selected.avatar as Avatar)" size="3xs" class="mx-0.5" />
    </template>
  </USelectMenu>
</template>
```


Or am I missing something?